src/engine/SimpleJson.ts

```diff
@@ -1,33 +1,49 @@
 export class SimpleJson {
   public static TextToDictionary(
-    text: string,
-    aggressiveFloatParsing: boolean = false
```

Comment:
Don't you want to keep the option around for Safari?

Reply:
I chose the lesser of two evils :
- either the math is totally wrong on safari unless aggressiveFloatParsing is explicit
- or there's a very minuscule chance of ", 5.0" being displayed ", 5.0f" in a text somewhere

Reply:
By essence, you never know where your code will be executed (Safari or elsewhere) so you basically never have the choice to turn the aggressiveFloatParsing. It is now on by default for Safari. And for everyone else, I use your suggestion.

Reply:
I read to fast and missed the fact that you only removed the option but kept the aggressive code. 👍